src/applications/coronavirus-screener/containers/App.jsx

```diff
@@ -31,6 +32,25 @@ export default function App({ params }) {
     alternateLangauge = 'en';
   }
 
+  // If there is a color indicated, store that for the pass results screen. Customization for Bronx VAMC.
+  let passFormResultsColors = {
```

Comment:
This chunk of code can be simplified (or complicated, depending on how you look at it 😆 ) like:
```
const defaultColorSpec = {
    background: '#112e51',
    font: 'white',
    name: '',
  };
  const colorKeys = Object.keys(passFormResultsColorOptions);

  const color = colorKeys.filter(
    clr => (location.href.toLowerCase().includes(clr) ? clr : null),
  );
  const passFormResultsColors = color?.length
    ? passFormResultsColorOptions[color]
    : defaultColorSpec;
```

Reply:
ah i knew there was some fancy JS shorthand to do this :-)